engine-cockpit-test/src_test/ch/ivyteam/enginecockpit/monitor/performance/TstBpmEngineManager.java

```diff
@@ -3,7 +3,7 @@
 import java.util.List;
```

Comment:
As this is repo is public now we should not do such strange things here ... Is it not possible to test this different than implementing an own IBpmEngineManager?